lib/plugins/standalone/index.js

```diff
@@ -3,14 +3,17 @@
 const fs = require('fs');
 const os = require('os');
 const path = require('path');
-const streamPromise = require('stream-promise');
+const { pipeline } = require('stream');
+const { promisify } = require('util');
 const fse = require('fs-extra');
 const fetch = require('node-fetch');
 const currentVersion = require('../../../package').version;
 const isStandaloneExecutable =
   require('../../utils/isStandaloneExecutable') && process.platform !== 'win32';
 const standaloneUtils = require('../../utils/standalone');
 
+const pipelineAsync = promisify(pipeline);
```

Comment:
I think we can keep it named `pipeline` or  `piplinePromised` (it was _async_ already, so `pipelineAsync` is slightly confusing)

e.g. in other files convention of naming is as https://github.com/serverless/serverless/blob/d102a3984abfe5c014c4adafa8abf2ea8edfd336/lib/utils/eventuallyUpdate.js#L20